pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -384,9 +383,11 @@ func NewJobController(config *rest.Config, serverOption *options.ServerOption) *
 
 	// Setup Quota
 	if serverOption.QuotaEnabled {
-		dispatchedAWDemands, dispatchedAWs := cc.getDispatchedAppWrappers()
-		cc.quotaManager, _ = quotamanager.NewQuotaManager(dispatchedAWDemands, dispatchedAWs, cc.queueJobLister,
-			config, serverOption)
+		// TODO Disable this when merg
+		// dispatchedAWDemands, dispatchedAWs := cc.getDispatchedAppWrappers()
+		// cc.quotaManager, _ = quotaforestmanager.NewQuotaManager(dispatchedAWDemands, dispatchedAWs, cc.queueJobLister,
+		//	config, serverOption)
+		klog.Warning("Quta manager enabled, but this version does not yet support it!!!")
```

Comment:
I am not sure I fully understand your comment, can you please provide more feedback ?

Reply:
I believe the answer of this question is, MCAD. Line 390 expresses that `this version does not yet support it`, so I was wondering what you mean by `this` here. Is it the current MCAD version, or is it the version of the tests we have right now?. I know they both are the same since the tests ARE part of MCAD, so maybe the question is meaningless. For example, would a version of MCAD without Quota Manager pass these tests? What about a version of MCAD with Quota Manager but running the old tests? Does that make sense?

Also I just noticed: `Quta` should be changed to `Quota`.

Reply:
Nit: please fix the typo in log message. @metalcycling this == current MCAD version until quota-management code is merged in main

Reply:
@asm582, I have commented out the whole block, the log message is no longer in the codebase. I believe this would resolve all  the issues.